packages/mui-base/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -210,7 +210,7 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
           }
 
           return (
-            <React.Fragment key={mark.value}>
+            <React.Fragment key={index}>
```

Comment:
```suggestion
            <React.Fragment key={`${mark.value}-${index}`}>
```

Wouldn't it be better to use a combination of the two, to avoid depending only on the index?

Reply:
Probably not necessary, let's not complicate the logic.